spec/models/post_spec.rb

```diff
@@ -1330,6 +1330,33 @@
     end
   end
 
+  context "scopes" do
+    it "scopes unignored posts properly" do
+      reader = create(:user)
+
+      unread_post = create(:post)
+      read_post = create(:post)
+
+      unread_ignored = create(:post)
+      read_ignored = create(:post)
+
+      ignored_board = create(:board)
+      create(:post, board: ignored_board)
+
+      read_post.mark_read(reader)
+
+      unread_ignored.ignore(reader)
+      read_ignored.ignore(reader)
+      read_ignored.mark_read(reader)
+
+      ignored_board.ignore(reader)
+
+      expect(Post.count).to eq(5)
+      expect(Post.not_ignored_by(reader).count).to eq(2)
```

Comment:
```suggestion
```
I think this line is redundant with the next one, and if it ends up being false (e.g. we actually see 3), it'll obscure the actual error a bit, right?